src/Settings/Path.hs

```diff
@@ -146,6 +150,12 @@ packageDbDirectory :: Stage -> FilePath
 packageDbDirectory Stage0 = buildRootPath -/- "stage0/bootstrapping.conf"
 packageDbDirectory _      = "inplace/lib/package.conf.d"
 
+-- | ( During installtion) Path to package database directory of a given 'Stage'.
```

Comment:
What does "during installation" refer to? As I understand, this is used not only durnig installation. I think this comment needs to be expanded, perhaps, giving a couple of specific examples of the resulting paths.